packages/style-engine/class-wp-style-engine-processor.php

```diff
@@ -86,11 +87,12 @@ public function get_css( $options = array() ) {
 		}
 
 		// Build the CSS.
-		$css = '';
+		$css      = '';
+		$new_line = $options['prettify'] ? "\n" : '';
 		foreach ( $this->css_rules as $rule ) {
-			$css .= $rule->get_css();
+			$css .= $rule->get_css( $options['prettify'] ) . $new_line;
```

Comment:
Should we add `$new_line` only if `$options['pretify'] is true? This way we'll also be able to get rid of the `rtrim()` call in the line below.

```suggestion
			$css .= $rule->get_css( $options['prettify'] ) . $new_line;
			$css .= $options['prettify'] ) ? $new_line : '';
```